src/pykeen/models/unimodal/hole.py

```diff
@@ -71,20 +71,13 @@ def __init__(
             preferred_device=preferred_device,
             random_seed=random_seed,
             regularizer=regularizer,
+            # Initialisation, cf. https://github.com/mnick/scikit-kge/blob/master/skge/param.py#L18-L27
+            entity_initializer=xavier_uniform_,
+            relation_initializer=xavier_uniform_,
+            entity_constrainer=clamp_norm,
```

Comment:
should we switch this to in-place?

Reply:
I thought about doing so, but I think it stays more flexible when we allow non in-place (out-place?) operations, and explicitly ensure that the tensor is changed in-place in the model.

Reply:
okay. we just have to keep straight that initializer functions have to be in place, while constrainer and normalizer ones have to be out place

Reply:
The init does not need to be in-place, in case we take care of operating in-place, cf. the implementation of torch.nn.init.*, e.g. https://pytorch.org/docs/stable/_modules/torch/nn/init.html#uniform_